doc/classes/Image.xml

```diff
@@ -762,5 +766,17 @@
 		<constant name="COMPRESS_SOURCE_NORMAL" value="2" enum="CompressSource">
 			Source texture (before compression) is a normal texture (e.g. it can be compressed into two channels).
 		</constant>
+		<constant name="BUMP_TO_NORMAL_ALGORITHM_PARTIAL_DERIVATIVE" value="0" enum="BumpMapToNormalMapAlgorithm">
+			Calculate normals in [code]bump_map_to_normal_map[/code] using the [url=https://en.wikipedia.org/wiki/Partial_derivative]partial derivatives[/url] of the heights. Faster, but potentially noisy.
+		</constant>
+		<constant name="BUMP_TO_NORMAL_ALGORITHM_SOBEL" value="1" enum="BumpMapToNormalMapAlgorithm">
+			Calculate normals in [code]bump_map_to_normal_map[/code] using a [url=https://en.wikipedia.org/wiki/Sobel_operator]Sobel operator[/url]. Slower, but smoother since it looks at more adjacent pixels than [code]BUMP_TO_NORMAL_ALGORITHM_PARTIAL_DERIVATIVE[/code].
+		</constant>
+		<constant name="BUMP_TO_NORMAL_WRAP_REPEAT" value="0" enum="BumpMapToNormalMapWrap">
+			Repeat wrapping in [code]bump_map_to_normal_map[/code]. Should be used with tileable textures to get seamlessly tiling normal maps.
+		</constant>
+		<constant name="BUMP_TO_NORMAL_WRAP_CLAMP" value="1" enum="BumpMapToNormalMapWrap">
+			Clamp wrapping in [code]bump_map_to_normal_map[/code]. Should always be used with non-tileable textures to prevent wrong normals at the border.
```

Comment:
```suggestion
			Clamp wrapping in [method bump_map_to_normal_map]. Should always be used with non-tileable textures to prevent wrong normals at the border.
```